templates/openstack/install/build-openstack.html

```diff
@@ -212,7 +210,7 @@ <h3 class="p-list-step__title"><span class="p-list-step__bullet">10</span>Config
           <div class="col-12">
             <h2>Need more help?</h2>
             <p>Let our cloud experts help you take the next step.</p>
-            <p><a href="/cloud/contact-us?product=cloud-build-openstack">Contact us&nbsp;&rsaquo;</a></p>
+            <p><a href="/openstack/contact-us?product=openstack-install-build">Contact us&nbsp;&rsaquo;</a></p>
```

Comment:
I think this might look better as a button... removed the &nbsp;&rsaquo; and add class p-button to the a href